plugin-gradle/src/main/java/com/diffplug/gradle/spotless/JavaExtension.java

```diff
@@ -207,13 +208,19 @@ public GoogleJavaFormatConfig reflowLongStrings(boolean reflowLongStrings) {
 			return this;
 		}
 
+		public GoogleJavaFormatConfig reorderImports(boolean reorderImports) {
+			this.reorderImports = reorderImports;
```

Comment:
it think a 
```
replaceStep(createStep());
```
is missing here, see https://github.com/diffplug/spotless/issues/1871

Reply:
Fixed in #1872.